requirements.yml

```diff
@@ -6,7 +6,7 @@ dependencies:
   - jinja2>=2.10
   - jupyter_client>=6.1
   - jupyter_core>=4.6.0
-  - notebook>=6.1.0rc1
+#  - notebook>=6.1.0rc1
```

Comment:
Please remove or add a reason why its being commented out - presumably to remember to add it back (and delete the `pip` section) once NB 6.1 is available - so that kind of thing should be mentioned.